apis/binding/v1alpha1/servicebinding_types.go

```diff
@@ -28,73 +28,81 @@ var templates = map[string]string{
+	// NamingStrategy defines custom string template for preparing binding
+	// names.  It can be set to pre-defined strategies: `none`,
+	// `lowercase`, or `uppercase`.  Otherwise, it is treated as a custom
+	// go template, and it is handled accordingly.
 	// +optional
 	NamingStrategy string `json:"namingStrategy,omitempty"`
 
-	// Custom mappings
+	// Mappings specifies custom mappings.
 	// +optional
 	Mappings []Mapping `json:"mappings,omitempty"`
 
-	// Services is used to identify multiple backing services.
+	// Services indicates the backing services to be connected to by an
+	// application.  At least one service must be specified.
 	// +kubebuilder:validation:MinItems:=1
 	Services []Service `json:"services"`
 
-	// Application is used to identify the application connecting to the
-	// backing service operator.
+	// Application identifies the application connecting to the backing
+	// service.
 	Application Application `json:"application"`
 
-	// DetectBindingResources is flag used to bind all non-bindable variables from
-	// different subresources owned by backing operator CR.
+	// DetectBindingResources is a flag that determines if the subresources
+	// of a backing operator custom resource will be bound or not.
 	// +optional
 	DetectBindingResources bool `json:"detectBindingResources,omitempty"`
 
-	// BindAsFiles makes available the binding values as files in the application's container
-	// See MountPath attribute description for more details.
+	// BindAsFiles makes the binding values available as files in the
+	// application's container.  See the MountPath attribute description
+	// for more details.
 	// +optional
 	// +kubebuilder:default:=true
 	BindAsFiles bool `json:"bindAsFiles"`
 }
 
-// ServiceBindingMapping defines a new binding from set of existing bindings
+// ServiceBindingMapping defines a new binding from a set of existing bindings.
 type Mapping struct {
-	// Name is the name of new binding
+	// Mapping is the name of new binding.
 	Name string `json:"name"`
-	// Value is a template which will be rendered and ibjected into the application
+	// Value specificies a template which will be rendered and injected
+	// into the application.
 	Value string `json:"value"`
 }
 
-// ServiceBindingStatus defines the observed state of ServiceBinding
+// ServiceBindingStatus defines the observed state of ServiceBinding.
 // +k8s:openapi-gen=true
 type ServiceBindingStatus struct {
-	// Conditions describes the state of the operator's reconciliation functionality.
+	// Condisions describes the state of the operator's reconciliation
+	// functionality.
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=type
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
-	// Secret is the name of the intermediate secret
+
+	// Secret indicates the name of the intermediate secret.
```

Comment:
I would rather use here `binding` instead of `intermediate`

Reply:
Done in 71484a9.